landsites_tools/settings.txt

```diff
@@ -3,11 +3,9 @@
 #####################################################################
 
 [user]
-sites2run       = ALP1, ALP2
-start_date      = 2010-01-01 00:00 # YYYY-MM-DD hh:mm
-end_date        = 2010-12-31 00:00 # YYYY-MM-DD hh:mm
+sites2run       = ALP1, SUB1
 run_type        =
-vegetation_type =                        # separate file 
+vegetation_type =                        # separate file
 atm_forcing     =
 archive_dir     =
 machine         =
```

Comment:
Should we rename the whole "user" section to "model_params" or, perhaps, "simulation_parameters" (to avoid confusion with surface paramerters)?

Reply:
Good suggestion!